addons/spreadsheet/static/src/global_filters/plugins/global_filters_ui_plugin.js

```diff
@@ -327,11 +328,11 @@ export default class GlobalFiltersUIPlugin extends spreadsheet.UIPlugin {
      * @returns {Domain|undefined}
      */
     _getDateDomain(filter, fieldMatching) {
-        if (!this.isGlobalFilterActive(filter.id)) {
-            return undefined;
-        }
         let granularity;
         const value = this.getGlobalFilterValue(filter.id);
+        if (!value || !fieldMatching.chain) {
+            return undefined;
```

Comment:
Shouldn't it return an empty domain instead ?
`_getDateDomain`'s `@returns` docstring is inconsistent with its caller's docstring (`getGlobalFilterDomain`) which tells it always returns a domain

Reply:
makes more sense yup